### PR TITLE
Only show the back button if it needed.

### PR DESCRIPTION
**Pull request by abhigyank**
Need review.

Everything works well, except a small issue described [here](https://chat.zulip.org/#narrow/stream/16-electron/topic/zulip-electron.20.2F.20master).

Address [issue 421](https://github.com/zulip/zulip-electron/issues/421)

### CODE REVIEW

**Comment by abhigyank** [comment]
@zulipbot label "needs review"

**Comment by priyank-p** [comment]
@abhigyank you need to used [`did-navigate-in-page`](https://electronjs.org/docs/api/webview-tag#event-did-navigate-in-page) and also [`did-navigate`](https://electronjs.org/docs/api/webview-tag#event-did-navigate) too event of web-view to track of the state, to know weather its time to update the button or not checking it on first time does work.

Edit: Just don't use page-title-update event.

File: app/renderer/js/components/webview.js [hunk 1]
```diff
@@ -69,6 +69,17 @@ class WebView extends BaseComponent {
 			this.props.onTitleChange();
 		});
 
+		this.$el.addEventListener('did-navigate-in-page', event => {
+			if (event.url.includes('renderer/preference.html')) {
```
Comment (akashnimare) [inline app/renderer/js/components/webview.js]: We could use it like - 
```
const isSettingPage = event.url.includes('renderer/preference.html');
if (isSettingPage) {
   return;
}
```

**Review by akashnimare — CHANGES_REQUESTED** [review]

**Review by akashnimare — APPROVED** [review]

**Comment by akashnimare** [comment]
Merged. Thanks, @abhigyank.

**Pull request merged**